slo-monitor/Makefile

```diff
@@ -0,0 +1,22 @@
+OUT_DIR = build
+PACKAGE = k8s.io/perf-tests/slo-monitor
+TAG = 0.5
+REPOSITORY = gmarek-kubernetes
+
+all: slo-monitor
+
+deps:
+	go get -u github.com/tools/godep
+
+slo-monitor: src/monitors/pod_monitor.go src/monitors/util.go src/monitors/watcher.go src/config/config.go src/main/slo-monitor.go deps
```

Comment:
Do we really need it? What's the cost of building it every time? We generally don't do it in other makefiles.